config/software/datadog-agent.rb

```diff
@@ -25,11 +25,15 @@
   mkdir  "#{install_dir}/agent/"
 
   # Agent code
+  mkdir "#{install_dir}/agent/checks.d"
   copy 'checks.d', "#{install_dir}/agent/"
+
   copy 'checks', "#{install_dir}/agent/"
   copy 'dogstream', "#{install_dir}/agent/"
   copy 'utils', "#{install_dir}/agent/"
 
+  copy 'requirements.txt', "#{install_dir}/agent/"
```

Comment:
Yes, for the constraints file, since we have to install the agent as part of building the individual packages.